FRANKDOC.md

```diff
@@ -18,13 +19,11 @@ The Frank!Doc provides reference information for Frank developers as explained i
 
 Here is more detailed information about some of these tags and annotations:
 
-**Annotation @FrankDocGroup:**: Has fields `name` and `order` (integer), which is used by the webapp to sort the groups. You can define a group by setting multiple `@FrankDocGroup` annotations with the same `name`. Only one `@FrankDocGroup` annotation of a group should have its `order` field set to avoid ambiguity.
+**Annotation @FrankDocGroup:**: Has fields `name` and `order` (integer), which is used by the webapp to sort the groups. You can define a group by setting multiple `@FrankDocGroup` annotations with the same `name`. Only one `@FrankDocGroup` annotation of a group should have its `order` field set to avoid ambiguity. This annotation behaves differently on classes and interfaces. When a Java class implements an interface that has a `@FrankDocGroup` annotation, then the class is put in the group non-exclusively. A class can belong to multiple groups when it implements multiple interfaces with different groups. However, when a class has a `@FrankDocGroup` annotation, then the class only belongs to that group. `@FrankDocGroup` annotations on classes are inherited by descendant classes, so descendant classes are also in the specified group exclusively.
```

Comment:
Ik denk dat het overzichtelijker is om dit te splitsen in een '@FrankDocGroup on a interface' en een '@FrankDocGroup on a class'

Reply:
Gedaan.